backend/src/controllers/serviceControllers.js

```diff
@@ -0,0 +1,46 @@
+import githubService from '../thirdParties/github/githubService';
+
+const getAppropriateService = serviceName => {
+  if (serviceName === 'github') {
+    return githubService;
+  }
+  return githubService;
+};
+
+const getServices = (req, res) => {
+  try {
+    const services = ['github'];
```

Comment:
why try/catch here ?it's just an assignment

Reply:
This file change is part of [this PR](https://github.com/pesto-students/batch-11-mernZapp/pull/34/files), so I have made requested change in that PR. So once that PR is merged, I will rebase it and that would reflect the changes.